src/utilities/index.ts

```diff
@@ -60,5 +60,34 @@ export const FormatBody = (message: string) => {
 }
 
 export function convertTimeStamp(timeStamp: string) {
-  return format(new Date(Number(timeStamp) * 1000), 'dd MMM yyyy | hh:mm a')
+  return format(new Date(Number(timeStamp) * 1000), "dd MMM yyyy | hh:mm a");
+}
+
+export async function httpRequest(url: RequestInfo, options?: RequestInit) {
+  const { body, ...customConfig } = options ?? {}
+  const headers = { 'Content-Type': 'application/json' }
+  const config: RequestInit = {
+    method: options?.method,
+    ...customConfig,
+    headers: {
+      ...body ? headers : {},
+      ...customConfig.headers,
+    },
+  }
+
+  if (body) {
+    config.body = JSON.stringify(body)
+  }
+
+  let data
```

Comment:
also i think `data` needs to be typed. it might have both success & error scenario types.